frontend/src/old-pages/Configure/Configure.tsx

```diff
@@ -301,22 +308,26 @@ function Configure() {
   useWizardSectionChangeLog()
 
   return (
-    <Layout contentType="form">
-      <StopDialog clusterName={clusterName} />
-      <SpaceBetween direction="vertical" size="l">
-        <BreadcrumbGroup
-          items={[
-            {text: 'Clusters', href: '#clusters'},
-            // @ts-expect-error TS(2741) FIXME: Property 'href' is missing in type '{ text: string... Remove this comment to see the full error message
-            {text: editing ? 'Update' : 'Create'},
-          ]}
+    <Layout
+      contentType="form"
+      breadcrumbs={
+        <Breadcrumbs
           onClick={clearStateAndCloseWizard}
+          pages={[
+            breadcrumbItemFromSlug('clusters'),
+            breadcrumbItem(editing ? 'Update' : 'Create'),
+          ]}
         />
+      }
+    >
+      {/*<Container >*/}
+      <StopDialog clusterName={clusterName} />
+      <SpaceBetween direction="vertical" size="l">
```

Comment:
question: why are you using two nested `SpaceBetween`? I'm struggling to understand the behavior here

Reply:
I don't know why GH shows it as added by this PR, but I actually didn't add any `SpaceBetween` components
This is as it was before so I wouldn't be able to answer :-/